test/controllers/questionnaires_controller_test.rb

```diff
@@ -76,6 +76,15 @@ class QuestionnairesControllerTest < ActionController::TestCase
       end
     end
 
+    context "with block questionnaires set" do
+      should "not allow creation" do
+        HackathonConfig['accepting_questionnaires'] = false
+        assert_difference('Questionnaire.count', 0) do
+          post :create, params: { questionnaire: { experience: @questionnaire.experience, interest: @questionnaire.interest, first_name: @questionnaire.first_name, last_name: @questionnaire.last_name, phone: @questionnaire.phone, level_of_study: @questionnaire.level_of_study, date_of_birth: @questionnaire.date_of_birth, shirt_size: @questionnaire.shirt_size, school_id: @school.id, agreement_accepted: "1", code_of_conduct_accepted: "1", data_sharing_accepted: "1", major: @questionnaire.major, gender: @questionnaire.gender, why_attend: @questionnaire.why_attend, graduation_year: @questionnaire.graduation_year, race_ethnicity: @questionnaire.race_ethnicity } }
```

Comment:
At some point could would you be open to refactoring all these JSON lines to not be on a single line?

Reply:
Possibly, I haven't looked into them much but if they provide better readability, why not?

Reply:
I wasn't sure if it was a ruby style thing -- in #241 I did it alongside the PR but since we have #335 (already refactoring tests) I think we can just wrap those changes under that issue, unless you think 2 PRs would be better.

Reply:
Agreed, let's wrap in #335.